assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -201,6 +201,10 @@ const ContextMenu = ( props ) => {
 		left: clientX,
 	};
 
+	if ( isRTL ) {
+		POPOVER_PROPS.position = 'bottom right';
```

Comment:
This is directly mutating the "constant", which is definitely not recommended, as it will change the value of something that looks to be constant. Sure, RTL won't suddenly change in the middle of the app, but still, it's bad practice.

Either, create two constants (re-using the common prop) and select which to use based on `isRTL` as:

```js
const popoverProps = isRTL ? POPOVER_PROPS_RTL : POPOVER_PROPS_LTR;
```

Or keep only the `className` as a constant:

```js
const popoverProps = {
	position: isRTL ? 'bottom right' : 'bottom left',
	className = POPOVER_CLASSNAME,
};
```

Maybe the latter is the purer option, but any like this works for me.

Reply:
@barklund What do you think of https://github.com/ampproject/amp-wp/pull/3633/commits/e960c73138f6abd823a0736e62abdb224e79efa9 ?